pkg/utils/build/quay.go

```diff
@@ -103,3 +104,35 @@ func DoesTagExistsInQuay(imageURL string) (bool, error) {
 	}
 	return false, nil
 }
+
+func IsImageRepoPublic(quayImageRepoName string) (bool, error) {
+	isPublic, err := quayClient.IsRepositoryPublic(quayOrg, quayImageRepoName)
+	if isPublic {
+		return true, nil
+	} else if !isPublic && err == nil {
+		return false, nil
+	} else {
+		return false, err
+	}
+}
+
+func IsQuayOrgSupportsPrivateRepo() (bool, error) {
```

Comment:
Is there any API to get current plan?

Reply:
I didn't find any API [here](https://docs.quay.io/api/swagger/) to get the current plan details